tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -53,7 +53,7 @@ public function getGenerateIndexSql(): string
     protected function getGenerateForeignKeySql(): string
     {
         return 'ALTER TABLE test ADD FOREIGN KEY (fk_name_id)'
-            . ' REFERENCES other_table (id) NOT DEFERRABLE INITIALLY IMMEDIATE';
+            . ' REFERENCES other_table (id)';
```

Comment:
`NOT DEFERRABLE INITIALLY IMMEDIATE` is the default, so not generating it is backward-compatible.